src/components/navpills/Navpills.jsx

```diff
@@ -1,8 +1,16 @@
 import { Link } from "react-router-dom";
+import { useCart } from "../../contexts/cart-context/cart-context";
+import { useLogin } from "../../contexts/login-context/login-context";
 
 export default function Navpills() {
-  //   const { state: cartState } = useCart();
+  const { stateCart, dispatchCart } = useCart();
+  const { dispatchUser } = useLogin();
   //   const { state: wishlistState } = useWishlist();
+  const signoutUser = () => {
+    localStorage.removeItem("userToken");
+    dispatchCart({ type: "RESET_CART" });
+    dispatchUser({ type: "LOGOUT_USER" });
+  };
```

Comment:
This function can be moved to context. Destructure the call and add it in onClick

Reply:
made separate login util for user signup/login functions